gazebo_ros2_control:
  ros__parameters:
    update_rate: 200
    use_sim_time: true

controller_manager:
  ros__parameters:
    update_rate: 200
    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster

    front_steer_controller:
      type: position_controllers/JointGroupPositionController

    front_drive_controller:
      type: velocity_controllers/JointGroupVelocityController

    rear_drive_controller:
      type: velocity_controllers/JointGroupVelocityController

front_steer_controller:
  ros__parameters:
    joints: [front_left_steer_joint, front_right_steer_joint]

front_drive_controller:
  ros__parameters:
    joints: [front_left_wheel_joint, front_right_wheel_joint]

rear_drive_controller:
  ros__parameters:
    joints: [rear_left_wheel_joint, rear_right_wheel_joint]
